src/dfx/src/util/assets.rs

```diff
@@ -83,3 +83,36 @@ pub fn assets_wasm(logger: &slog::Logger) -> DfxResult<Vec<u8>> {
     }
     Ok(wasm)
 }
+
+#[allow(unused)]
+#[context("Failed to load bitcoin wasm.")]
+pub fn bitcoin_wasm(logger: &slog::Logger) -> DfxResult<Vec<u8>> {
+    let mut wasm = Vec::new();
+
+    if let Ok(dfx_assets_wasm) = std::env::var("DFX_BITCOIN_WASM") {
+        info!(logger, "Using wasm at path: {}", dfx_assets_wasm);
+        std::fs::File::open(&dfx_assets_wasm)
+            .with_context(|| format!("Failed to open {}.", dfx_assets_wasm))?
+            .read_to_end(&mut wasm)
+            .with_context(|| format!("Failed to read file content for {}.", dfx_assets_wasm))?;
```

Comment:
Please use dfx_core methods when available:

```suggestion
        wasm = dfx_core::fs::read(&PathBuf::from(&dfx_assets_wasm))?;
```